docs/papers-publication/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: papers-publications
+show_sidebar: false
+permalink: /papers-publications/
+
+# Banner Content
+# =================================
+title: Papers and Publications
+subtitle: We have been involved in these and related research areas for several years
+hero_image: /assets/img/home-callouts/papers_publications.jpg
+
+# Papers and Publications list
+# =================================
+publications:
+  - title: Machine-based detection and classification for bone marrow aspirate differential counts - initial development focusing on nonneoplastic cells.
+    full-citation: "Chandradevan, Ramraj, Ahmed A. Aljudi, Bradley R. Drumheller, Nilakshan Kunananthaseelan, Mohamed Amgad, David A. Gutman, Lee A. D. Cooper, and David L. Jaye. “Machine-Based Detection and Classification for Bone Marrow Aspirate Differential Counts: Initial Development Focusing on Nonneoplastic Cells.” Laboratory Investigation 100, no. 1 (September 30, 2019): 98–109. https://doi.org/10.1038/s41374-019-0325-7"
```

Comment:
Can we make the https://doi.org clickable links?

Reply:
Fixed in 9f2141ab99162151e586eccf607085185d41f508